website/src/data/users.js

```diff
@@ -476,6 +476,14 @@ const users = [
     source: 'https://github.com/linyuxuanlin/Wiki_Docusaurus',
     tags: ['large', 'personal'],
   },
+  {
+    title: 'PptxGenJS',
+    description: 'PowerPoint JavaScript Library',
+    preview: require('./showcase/pptxgenjs.png'),
+    website: 'https://gitbrent.github.io/PptxGenJS/',
+    source: 'https://github.com/gitbrent/PptxGenJS/tree/gh-pages',
```

Comment:
can you change this source url to the repo where the Docusaurus site sources are? (I mean, not the built HTML files)